docs/DeveloperGuide.md

```diff
@@ -350,6 +354,54 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: UC06 - Create an event**
+
+**Guarantees: MSS -> Entered event will be added.**
+
+**MSS**
+
+1.  User requests to add an event to the list
+2.  CCACommander adds that event to the event list and displays the event added
+
+* 1a. User uses the command incorrectly
+
+    * 1a1. CCACommander shows an error message
+
+      Use case ends.
+  
+* 1b. The entered event already exists
+
+    * 1b1. CCACommander shows an error message
+
+      Use case ends.
+
+**Use case: UC07 - Delete an event**
+
+**Guarantees: MSS -> Specified event will be deleted.**
+
+**MSS**
+
+1.  User <ins>lists all events</ins> (UC02)
+2.  CCACommander shows a list of all events
+3.  User requests to delete a specific event in the list of all events
+4.  CCACommander deletes the specific event
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list of all events is empty.
+
+    * 2a1. CCACommander shows an error message.
+
+      Use case ends.
```

Comment:
I think this can be removed because an empty list is not an error. If the user tries to delete an event when list is empty, the case is also covered under the invalid index.

```suggestion
```